source/codegen/metadata/nirfmxlte/config.py

```diff
@@ -1,16 +1,16 @@
 # -*- coding: utf-8 -*-
 config = {
```

Comment:
Oops. I'd rather not reformat these. But unlike the functions.py files, they're not in strict "pretty print" format in the first place. So maybe it's OK. I'm going to try to fix it.